packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts

```diff
@@ -197,6 +227,16 @@ export default class AddFragment extends BaseDialog {
         this.model.setProperty('/selectedIndex', indexArray.length - 1);
         this.model.setProperty('/targetAggregation', controlAggregation);
         this.model.setProperty('/index', indexArray);
+
+        this.popover = new Popover({
```

Comment:
Why could not this be included in the `*.view.xml` file? We should not import ui5 controls directly

Reply:
resolved in [e94a899](https://github.com/SAP/open-ux-tools/pull/1477/commits/e94a89919304cadb80bf98ae1eb4ece1069cc8cb)